config/Config-kernel.in

```diff
@@ -1418,8 +1418,7 @@ config KERNEL_JFFS2_FS_SECURITY
 
 config KERNEL_WERROR
 	bool "Compile the kernel with warnings as errors"
-	default BUILDBOT
-	default y if GCC_USE_VERSION_12
+	default y
```

Comment:
This change equals to [CONFIG_WERROR=y](https://github.com/openwrt/openwrt/commit/b2d1eb717b6597e0145c98fc5f9f88a7dc219513) which we had initially, thus we're going to get plethora of following bug reports soon https://github.com/openwrt/openwrt/issues/12687 https://github.com/openwrt/openwrt/issues/12622 https://github.com/openwrt/openwrt/issues/12734